src/sentry/static/sentry/app/components/panels/panelItemGroup.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'react-emotion';
+
+import PanelItem from './panelItem';
+
+const PanelItemGroup = styled('div')`
+  border-left: 4px solid ${p => p.theme.borderLight};
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  background: ${p => p.theme.offWhite};
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  ${PanelItem} {
+    border-bottom-color: ${p => p.theme.borderLight};
+
+    &:last-child {
```

Comment:
if you want to up your wizard quotient you can do `&:not(:last-child) { border-bottom-color: blah };` and skip the override.

Reply:
I would, but the PanelItem has the border, so I have to override it.

Reply:
makes sense!